api/v3/openapi.yaml

```diff
@@ -2725,6 +2952,28 @@ components:
                 partition_reassignments:
                   related: 'http://localhost:9391/v3/clusters/cluster-1/topics/topic-3/partitions/-/reassignments'
 
+    ProduceResponse:
+      description: ''
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ProduceResponse'
+          example:
+            cluster_id: 'cluster-1'
+            topic_name: 'topic-1'
+            partition_id: 1
+            offset: 0
+            timestamp: '2021-02-05T19:14:42Z'
```

Comment:
nit: Is the timestamp format always this? AFAIR a server-prodived `Instant` is the source of the timestamp, and it gets formatted to a human-readable string somewhere in Kafka REST.

Reply:
See https://github.com/confluentinc/kafka-rest/blob/master/kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestApplication.java#L129.